src/js/header.js

```diff
@@ -0,0 +1,14 @@
+const header = document.querySelector('.header');
+let isScrolled = false;
+
+window.addEventListener('scroll', () => {
+  const scrollTop = window.scrollY || document.documentElement.scrollTop;
+
+  if (scrollTop >= 2 && !isScrolled) {
+    header.classList.add('header-scrolled');
```

Comment:
Do we have a `header-scrolled` class? If not, please add it

Reply:
yes, sorry, did not push latest version